### ISSUE TITLE
Admin section extremely confusing

### ISSUE THREAD

**Issue by canibanoglu** [netlify/gotrue-js#47]
In this section of the documentation there is a lot of mentions of admin methods that are not exposed (how does that even work and why mention them if they are not exposed?) and a lot of example code that makes absolutely no use of the library methods. Instead we see how to get the token from the lambda function context and manually make calls to various endpoints. 

The information is certainly helpful but it should be part of the Identity and Functions documentations as it just creates confusion on this repo's documentation. Or maybe something went wrong and the wrong information is displayed in the code examples as the mentioned methods are nowhere to be seen in the example code.

**Comment by sw-yx** [comment]
thanks canibanoglu. indeed the docs here are not well maintained. they probably shouldnt live in the identity and functions docs as gotruejs is lower level. happy to take PRs, for the time being this isn't a high priority for me, but happy to work with you on any netlify internal questions you may need answered. the code isn't too bad to go through.

**Comment by canibanoglu** [comment]
Alright, I will go through the code and try to come up with some updates to the documentation, should be good for me to internalize gotrue as well :)

**Comment by sw-yx** [comment]
thanks very much. :(

**Comment by QuantumInformation** [comment]
Hey @sw-yx have you had much luck with the User.update method? My issue is blocking me making more videos.

https://github.com/netlify/gotrue-js/issues/65

**Comment by sw-yx** [comment]
no, sorry. for the record i dont maintain this library.

**Comment by QuantumInformation** [comment]
Yeah, I can see that, I think I will have to find another authentication lib as this doesn't appear to be actively maintained atm.
